### PR TITLE
Adds analytics

### PR DESCRIPTION
**Pull request by andrewlew1s**
We are using firebase analytics which you can navigate to through the analytics part of the console.

https://console.firebase.google.com/u/0/project/holistic-toolbox/analytics/app/web:OTBjZmUzYjEtNGI1Yi00MDE0LWFhZTItN2EyZmQ1ZDVkMGZk/overview~2F%3Ft=1579048983265&fpn=94330098957&swu=1&sgu=1&sus=upgraded&cs=app.m.dashboard.overview&g=1

These are the docs I followed:
https://firebase.google.com/docs/analytics/get-started?platform=web&authuser=0

We added the necessary firebase code to index.js of tools-layout.

### CODE REVIEW

File: tools/json-formatter/package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
 		"bootstrap": "^4.4.1",
 		"bootstrap-vue": "^2.1.0",
 		"core-js": "^3.4.4",
+		"firebase": "^7.6.2",
+		"marked": "^0.8.0",
```
Comment (michaelfitzhavey) [inline tools/json-formatter/package.json]: marked?

File: tools/json-formatter/src/App.vue [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <template>
 	<div id="app">
+		<analytics />
```
Comment (michaelfitzhavey) [inline tools/json-formatter/src/App.vue]: what's this do?

File: tools/json-formatter/src/App.vue [hunk 3]
```diff
@@ -8,11 +9,13 @@
 
 <script>
 // import { ToolLayout } from 'toolbox-layout';
```
Comment (michaelfitzhavey) [inline tools/json-formatter/src/App.vue]: let's remove this commented out line to clean up

File: tools/json-formatter/src/components/JsonFormatter.vue [hunk 4]
```diff
@@ -57,7 +57,7 @@
 				rows="20"
 				disabled/>
 		</div>
-
+
```
Comment (michaelfitzhavey) [inline tools/json-formatter/src/components/JsonFormatter.vue]: if this is added white space let's remove it

File: tools/json-formatter/src/App.vue (hunk 2 above)
Comment (andrewlew1s) [reply tools/json-formatter/src/App.vue]: this was meant to have been deleted

**Pull request merged**